Cartfile

```diff
@@ -1,4 +1,4 @@
-binary "https://api.mapbox.com/downloads/v2/carthage/mapbox-common/MapboxCommon.json" == 21.2.0
-binary "https://api.mapbox.com/downloads/v2/carthage/mobile-navigation-native/MapboxNavigationNative.xcframework.json" ~> 94.0.3
+binary "https://api.mapbox.com/downloads/v2/carthage/mapbox-common/MapboxCommon.json" == 21.3.0
```

Comment:
From now on we pin to MapboxCommon `v21.3.0` in all package managers (SPM, Carthage and CocoaPods).

Reply:
We only support Carthage for MapboxCoreNavigation, which doesn’t pull in the map SDK at all. In principle, it’s only MapboxNavigation that needs to pin MapboxCommon due to its map SDK dependency. However, it would make sense to ensure consistency among package managers to the extent possible.

Reply:
I agree, resolved dependencies should be similar for different package managers.